### PR TITLE
Add a test for creating/deleting a CosmosDB Account

### PR DESCRIPTION
**Pull request by Porges**

### CODE REVIEW

File: hack/generated/controllers/crd_cosmosdb_test.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+	expectedKind := documentdb.DatabaseAccountStatusKindGlobalDocumentDB
+	g.Expect(*acct.Status.Kind).To(Equal(expectedKind))
+
+	g.Expect(acct.Status.Id).ToNot(BeNil())
+	armId := *acct.Status.Id
+
+	// Run sub-tests
+	/*
+		t.Run("Blob Services CRUD", func(t *testing.T) {
```
Comment (babbageclunk) [inline hack/generated/controllers/crd_cosmosdb_test.go:65]: Just left around from another test?

File: hack/generated/controllers/suite_test.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ import (
 
 const (
 	TestNamespace          = "k8s-infra-test-ns"
-	DefaultResourceTimeout = 2 * time.Minute
+	DefaultResourceTimeout = 10 * time.Minute
```
Comment (babbageclunk) [inline hack/generated/controllers/suite_test.go:22]: Is there a way to make this 10 min just for CosmosDB and keep other resources 2 min?

**Review by babbageclunk — APPROVED** [review]
Looks nice! Line count threw me for a moment. :)

**Review by theunrepentantgeek — APPROVED** [review]

File: hack/generated/controllers/suite_test.go (hunk 2 above)
Comment (Porges) [reply hack/generated/controllers/suite_test.go:22]: @babbageclunk we already had some timeouts on master with the live tests so perhaps better to make it 10 minutes for everything? That's about the maximum we can do, BTW, go kills the tests after 11 minutes 😆

**Pull request merged**